src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -127,3 +127,33 @@ Alpine
 ```sh
 apk add cmake openssl-dev openssl-libs-static
 ```
+
+## Using statically linked NUMA
+This feature can statically link NUMA library (libnuma.a) into your applications at build time.
+NativeAOT binaries built with this feature can run even when NUMA libraries are not installed.
+
+You can use this feature by adding the `StaticNumaLinking` property to your project file as follows:
+
+```xml
+<PropertyGroup>
+  <StaticNumaLinking>true</StaticNumaLinking>
+</PropertyGroup>
+```
+
+License (LGPL v2.1): https://github.com/numactl/numactl/blob/master/LICENSE.LGPL2.1. Note that this license imposes specific requirements on distribution of statically linked binaries.
+
+### Prerequisites
+
+Ubuntu
+```sh
+apt install libnuma-dev
+```
+
+Alpine
+```sh
+apk add numactl-dev
+```
+
+NixOS
+NativeAOT uses native executable `ilc` pulled from nuget, which has special requirements. Docs can be found at https://nixos.wiki/wiki/DotNET#NativeAOT
+
```

Comment:
```suggestion

## NixOS
NativeAOT uses native executable `ilc` pulled from nuget, which has special requirements. Docs can be found at https://nixos.wiki/wiki/DotNET#NativeAOT
```

Reply:
The paragraph about statically linked NUMA was deleted.

Reply:
Woops, thanks